src/bluebot/tools/interactive-tester.ts

```
@@ -0,0 +1,212 @@
+			}
+			return true;
+
+		case '/enemy':
+			const enemyId = process.env.BLUEBOT_ENEMY_USER_ID;
+			if (enemyId) {
+				identity.userId = enemyId;
+				console.log(`${colors.green}Switched to enemy user mode (ID: ${enemyId})${colors.reset}\n`);
+			} else {
+				console.log(`${colors.red}BLUEBOT_ENEMY_USER_ID not set in environment${colors.reset}\n`);
+			}
+			return true;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)